frames/src/main/java/com/checkout/frames/screen/manager/PaymentFormStateManager.kt

```diff
@@ -25,6 +29,21 @@ internal class PaymentFormStateManager : PaymentStateManager {
 
     override val country: MutableStateFlow<Country> = MutableStateFlow(Country.from(Locale.getDefault().country))
 
+    override val supportedCardSchemeList = provideCardSchemeList()
+
+    @VisibleForTesting
+    fun provideCardSchemeList(): List<CardScheme> =
+        providedSupportedCardSchemeList ?: listOf(
+            CardScheme.VISA,
+            CardScheme.MASTERCARD,
+            CardScheme.MAESTRO,
+            CardScheme.AMERICAN_EXPRESS,
+            CardScheme.DISCOVER,
+            CardScheme.DINERS_CLUB,
+            CardScheme.JCB,
+            CardScheme.MADA
+        )
```

Comment:
It would be nice to use a dynamic approach for generation of all card schemes list, just get all schemes for enum and remove Unknown.
Also, it would be nice to have that logic in the CardScheme enum as a public function, as it could be useful for a merchant.